pyproject.toml

```diff
@@ -46,3 +46,17 @@ source = [
 omit = [
     "models/*"
 ]
+
+[tool.mypy]
+follow_imports = "normal"
+ignore_missing_imports = true
+show_column_numbers = true
+show_error_codes = true
+no_implicit_optional = true
+warn_redundant_casts = true
+warn_unused_ignores = true
+#disallow_untyped_defs = true
+#disallow_untyped_decorators = true
```

Comment:
these two should be turned on now that everything is typed
we turned them off in eo-learn because we have some more typing issues to resolve